libraries/AP_Motors/AP_MotorsHeli_Single.cpp

```diff
@@ -466,10 +468,10 @@ void AP_MotorsHeli_Single::move_actuators(float roll_out, float pitch_out, float
         // the feed-forward is not required when the motor is stopped or at idle, and thus not creating torque
         // also not required if we are using external gyro
         if ((_main_rotor.get_control_output() > _main_rotor.get_idle_output()) && _tail_type != AP_MOTORS_HELI_SINGLE_TAILTYPE_SERVO_EXTGYRO) {
-            // sanity check collective_yaw_effect
-            _collective_yaw_effect.set(constrain_float(_collective_yaw_effect, -AP_MOTORS_HELI_SINGLE_COLYAW_RANGE, AP_MOTORS_HELI_SINGLE_COLYAW_RANGE));
+            // sanity check collective_yaw_scale
+            _collective_yaw_scale.set(constrain_float(_collective_yaw_scale, -AP_MOTORS_HELI_SINGLE_COLYAW_RANGE, AP_MOTORS_HELI_SINGLE_COLYAW_RANGE));
             // the 4.5 scaling factor is to bring the values in line with previous releases
```

Comment:
this comment looks like it's out-of-date now that the 4.5 is gone..

Reply:
fixed.  Thanks!